src/test/scala/com/mwz/sonar/scala/scapegoat/ScapegoatSensorSpec.scala

```diff
@@ -363,6 +379,62 @@ class ScapegoatSensorSpec
     // validate the sensor behavior
     sensorContext.allIssues shouldBe empty
   }
+
+  it should "not report module issues" in {
```

Comment:
I think that _'not'_ is out of place.

Reply:
thanks for spotting this - fixed in 61a613a